Base/QTCLI/qSlicerCLIModuleUIHelper.cxx

```diff
@@ -137,8 +137,12 @@ QButtonGroup* ButtonGroupWidgetWrapper::buttonGroup()const
 QString ButtonGroupWidgetWrapper::checkedValue()
 {
   QAbstractButton* button = this->ButtonGroup->checkedButton();
-  Q_ASSERT(button);
-  return button->text();
+  if (button) {
+    return button->text();
+  }
+  else {
```

Comment:
Thanks for the patch :+1: 

Nitpick: Could you add a comment like:

```
// Handle case where <string-enumeration> which has no <default> element
```

Thanks